### PR TITLE
Create export-pushers.md

### PR DESCRIPTION
**Pull request by nikija**

### CODE REVIEW

File: content/users/commander/export-pushers.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
+In order to set up a Pusher :
+
+1. Create a pusher configuration
+2. Choose a report that you would like to export and export it using a Pusher by clicking the OK button
+3. Proceed to Exports in the left-hand side menu and choose Pushers
+4. Choose the needed Pusher
+5. Set the settings according to your needs
+6. Check Report configuration settings
+7. Enable the reservation and click okay
```
Comment (jirihelmich) [inline content/users/commander/export-pushers.md]: "okay" is not there, is it?

File: content/users/commander/export-pushers.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+- Name - name of the pusher
+- Is enabled - ticked checkbox notifies the pusher is enabled
+- Period - specifies how often the report will be generated (e.g. once per day,  hour etc.)
+- Start - date and time the pushing starts
+- Pusher configuration - the dropdown that specifies pusher configurations you have set before
+- Options - the enabled Notify about export checkbox will send a notification in the Notifications sections on the top right of the screen.
+
+Report configuration
+
+- Start offset -  defines how long before your start period (specified in the Pusher) the start date of the report will be
```
Comment (jirihelmich) [inline content/users/commander/export-pushers.md]: two spaces after dash

File: content/users/commander/export-pushers.md [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+
+Example: If you would like to see a reservation report for the following week when you come to work on Monday at 9 am, you can set the Start offset to 9 hours (so that the report starts at the very beginning of the week) and set the End offset to 7 days (so that it finishes at the very end of the week).
+
+Settings: this field represents the settings you have specified at the time of pushing your report.
+
+Good to know:
+
+- Be careful when setting Period to Month. If you choose a monthly Period,  the reports you are pushing should be ending not later than on the 28th of the month. This restrictions comes in due to the month of February.
+
+- If you need a report to be generated for each full month, you can do it in Report configurations by setting an offset to one month.
```
Comment (jirihelmich) [inline content/users/commander/export-pushers.md]: Screenshot with an example would be nice.

File: content/users/commander/export-pushers.md [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+
+- If you would like a report to contain data for a period of one year, set the offset to 12 months.
```
Comment (jirihelmich) [inline content/users/commander/export-pushers.md]: I am missing some overall description of what the functionality can do for the user.

File: content/users/commander/export-pushers.md (hunk 3 above)
Comment (nikija) [reply content/users/commander/export-pushers.md]: I believe Matt was strongly against using screenshots in the guide, should I use for this one?

File: content/users/commander/export-pushers.md (hunk 4 above)
Comment (nikija) [reply content/users/commander/export-pushers.md]: Added, could you take a look please?

**Pull request merged**